server/middlewares/validators/videos/video-channels.ts

```diff
@@ -151,31 +149,8 @@ export {
 
 // ---------------------------------------------------------------------------
 
-function checkUserCanDeleteVideoChannel (user: MUser, videoChannel: MChannelAccountDefault, res: express.Response) {
-  if (videoChannel.Actor.isOwned() === false) {
```

Comment:
I think this `if` should be used to check all routes except `/:nameWithHost/followers`

You could put this part in a dedicated middleware, and call it in controllers

Reply:
Should we really use this check on regular GET routes?

Reply:
There's middleware called `localVideoChannelValidator` but it returns a 404 if the video channel isn't found on the local server. It's only used by activity pub routes, should I move that middleware to `validators/activitypub/video-channel.ts` and then create a new middleware in `validators/videos/video-channel.ts`?

Reply:
I think I'll remove `localVideoChannelValidator`
Thanks!